lib/src/Base/Stat/CovarianceMatrix.cxx

```diff
@@ -45,6 +45,13 @@ CovarianceMatrix::CovarianceMatrix(const MatrixImplementation & i)
   // Nothing to do
 }
 
+/* Constructor from SymmetricMatrix */
+CovarianceMatrix::CovarianceMatrix(const SymmetricMatrix & sym)
```

Comment:
Do we really need that? Isn't it supposed to be done by inheritance?

Reply:
Yes, I tried a the c++ level